rpcserver.go

```diff
@@ -510,17 +510,47 @@ func (s *rpcServer) InitAccount(ctx context.Context,
 			"must be specified")
 	}
 
-	// Determine the desired transaction fee.
-	confTarget := req.GetConfTarget()
-	if confTarget < 1 {
-		return nil, fmt.Errorf("confirmation target must be " +
-			"greater than 0")
+	var feeRate chainfee.SatPerKWeight
+
+	switch {
+	case req.GetFeeRateSatPerKw() != 0:
+		feeRate = chainfee.SatPerKWeight(req.GetFeeRateSatPerKw())
```

Comment:
nit: with longer `switch` statements we add a newline before each new `case` to give the eye some more orientation where a block ends.

Reply:
not compelte but here's a generic guide to explain our coding style: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#code-spacing